command/agent.go

```diff
@@ -259,6 +260,18 @@ Telemetry Options:
   -telemetry-circonus-broker-select-tag
     A tag which is used to select a broker ID when an explicit broker ID is not
     provided.
+  
+High Availability Options:
+  -high-availability
+	On cases when multiple instances of the autoscaler need to be run at the same
+	time, the high-availability option triggers a leader election using a lock 
+	for sync among the different lock instances. It defaults to false.
+  
+  -lock-path
+    When using the high-availability mode, the path to the lock to be used for the
+	leader election can be provided using the lock-path flag. The same path must 
+	be provided to every instance of the autoscaler in order to be included in the 
+	election.
```

Comment:
Other flags are suffixed with their component name and I wonder if we want to follow suit here? That would mean the flags become something like `high-availability-enabled` and `high-availability-lock-path`. I also think the flags names should mimic the config block names, to make it easier. for operators to understand.

Reply:
Done!